### DISCUSSION TITLE
Statically compiled bundle.go report parsing errors in goland

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4003] [Q&A]
![截屏2023-06-25 18 02 36](https://github.com/fyne-io/fyne/assets/1819457/a66cec7f-ce04-41f4-9322-61cc9ef1fb31)

Although it does not affect the use, is there a regular usage?

**Comment by andydotxyz** [comment]
It's not clear what the question is here. If the code compiles but you have an error in IDE then it's probably a local configuration problem.
